alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts

```diff
@@ -112,6 +115,12 @@ export class DecisionComponentComponent implements OnInit {
         case APPLICATION_DECISION_COMPONENT_TYPE.SUBD:
           this.patchSubdFields();
           break;
+        case APPLICATION_DECISION_COMPONENT_TYPE.INCL:
+          this.patchInclExclFields();
+          break;
+        case APPLICATION_DECISION_COMPONENT_TYPE.EXCL:
+          this.patchInclExclFields();
+          break;
```

Comment:
if the component props are the same you could do 
`      case APPLICATION_DECISION_COMPONENT_TYPE.INCL:
        case APPLICATION_DECISION_COMPONENT_TYPE.EXCL:
          this.patchInclExclFields();
          break;`

Reply:
done! Good suggestion.